mirobo/vacuum.py

```diff
@@ -22,13 +23,17 @@ def __init__(self, ip, token, debug=0):
         self.__id = 0
         self._devtype = None
         self._serial = None
+        self._ts = None
+        self._ts_server = int(time.mktime(datetime.datetime.utcnow().timetuple()))
```

Comment:
_ts_server indicates that it is a server timestamp, and if so, initializing it here sounds wrong? In anycase it's better to use datetime objects instead of converting it to an int.